debian/tests/integrationtests-spread

```diff
@@ -0,0 +1,45 @@
+#!/bin/sh
+
+set -ex
+
+# required for the debian adt host
+mkdir -p /etc/systemd/system/snapd.service.d/
+if [ "${http_proxy:-}" != "" ]; then
+	cat <<-EOF | tee /etc/systemd/system/snapd.service.d/proxy.conf
+		[Service]
+		Environment=http_proxy=$http_proxy
```

Comment:
would this be a harmless unit if $http_proxy is not defined?

Reply:
It wouldn't be defined at all (see the `if` statement above).

Reply:
duh, I completely missed that; my suggestion was going to be to gate it just like you did!